### ISSUE TITLE
Error occurred when executing KSampler: No module named 'ldm.models.diffusion.ddpm'

### ISSUE THREAD

**Issue by toyxyz** [ModelSurge/sd-webui-comfyui#108]
When using the Webui Save Image node, the first image generation works, but the following error occurs when generating images after that. If I don't use that node there is no problem.

commit : 9d01ada

----------------------------------------------

Error occurred when executing KSampler:

No module named 'ldm.models.diffusion.ddpm'

File "C:\ComfyUI_windows_portable\ComfyUI\execution.py", line 151, in recursive_execute
output_data, output_ui = get_output_data(obj, input_data_all)
File "C:\ComfyUI_windows_portable\ComfyUI\execution.py", line 81, in get_output_data
return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
File "C:\ComfyUI_windows_portable\ComfyUI\execution.py", line 74, in map_node_over_list
results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
File "C:\ComfyUI_windows_portable\ComfyUI\nodes.py", line 1206, in sample
return common_ksampler(model, seed, steps, cfg, sampler_name, scheduler, positive, negative, latent_image, denoise=denoise)
File "C:\ComfyUI_windows_portable\ComfyUI\nodes.py", line 1167, in common_ksampler
previewer = latent_preview.get_previewer(device, model.model.latent_format)
File "C:\ComfyUI_windows_portable\ComfyUI\latent_preview.py", line 65, in get_previewer
taesd = TAESD(None, taesd_decoder_path).to(device)
File "C:\ComfyUI_windows_portable\ComfyUI\comfy\taesd\taesd.py", line 55, in __init__
self.decoder.load_state_dict(torch.load(decoder_path, map_location="cpu", weights_only=True))
File "C:\stable-diffusion-webui\modules\safe.py", line 108, in load
return load_with_extra(filename, *args, extra_handler=global_extra_handler, **kwargs)
File "C:\stable-diffusion-webui\modules\safe.py", line 133, in load_with_extra
from modules import shared
File "C:\stable-diffusion-webui\modules\shared.py", line 21, in
from ldm.models.diffusion.ddpm import LatentDiffusion

![image](https://github.com/comfyanonymous/ComfyUI/assets/8006000/1110c3c8-b3e4-4f25-b677-c982d47372a2)

![image](https://github.com/comfyanonymous/ComfyUI/assets/8006000/12bea3de-5645-4102-bd5a-f18b93e61547)

**Comment by ljleb** [comment]
It seems the node somehow imports webui modules directly, which we do not want to happen because both environments have slightly different implementations of the ldm module.
